FirebaseCore/Tests/Unit/FIROptionsTest.m

```diff
@@ -627,7 +627,7 @@ - (void)testVersionFormat {
 }
 
 // TODO: The version test will break when the Firebase major version hits 10.
-- (void)testVersionConsistency {
+- (void)SKIP_testVersionConsistency {
```

Comment:
Fix these tests before merging. I've disabled them as to unblock staging.

Reply:
Done in c3470de. I could update the comment to:
```diff
- ... when the Firebase major version hits 10.
+ ... when the Firebase major version hits 100.
```
but I have a feeling this test won't be written in Objective-C by then 😜

Reply:
Resolving because c3470de fixed the tests: https://github.com/firebase/firebase-ios-sdk/actions/runs/3106850948/jobs/5034284561